src/nbl/video/CVulkanCommandBuffer.h

```diff
@@ -1003,6 +1030,12 @@ class CVulkanCommandBuffer : public IGPUCommandBuffer
 
     bool endRenderPass() override
     {
+        if (!emplace<IGPUCommandPool::CEndRenderPassCmd>())
+        {
+            __debugbreak();
```

Comment:
@AnastaZIuk what's out cross platform macro for this?

Reply:
No reason to put a `__debugbreak` here! It was just for testing, forgot to remove.

Reply:
ah ok.

Reply:
I think this can be resolved.